apps/exts/cesium.performance/config/extension.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+title = "Cesium for Omniverse Performance Testing Extension"
+version = "0.0.0"
+app = false
```

Comment:
We should add `toggleable = false`.

Reply:
Is there a reason for setting `toggleable = false`? I think it's fine if someone decides to disable the extension.

For example, `omni.code.app.setup` doesn't set `toggleable = false` either.

Reply:
I guess there's no harm in it. My fear is that if a user disables it and then reenables it, the startup stuff will execute again, but that may be harmless. Seems undesirable regardless.

Reply:
Alright, I added `toggleable = false`. I don't see much harm either way.